src/main/scala/temple/Main.scala

```diff
@@ -13,7 +15,7 @@ object Main extends App {
   try {
     val config = new TempleConfig(args)
     config.subcommand match {
-      case Some(config.Generate) => Application.generate(config)
+      case Some(config.Generate) => Application.generate(config, PoliceSergeantNicholasAngel)
```

Comment:
I still think this should be a mock and not the main application

Reply:
That was gonna come in a separate PR